republish_command.txt

```diff
@@ -0,0 +1 @@
+rosrun image_transport republish compressed in:=/camera/image_raw raw out:=/camera/image_raw
```

Comment:
this republishes `/camera/image_raw/compressed` as `/camera/image_raw`

this is necessary because rosbags (the ones we have) only have the compressed stream, and we need it uncompressed for the vision node